rerun_py/tests/e2e_redap_tests/resources/dataset/README.md

```diff
@@ -0,0 +1,104 @@
+## Purpose
+
+These RRDs are an example dataset for testing gRPC calls between rerun and the OSS server.
+
+## Generating
+
+We could either generate the dataset on the fly or we can commit RRDs to the repository.
+By generating on the fly we run the issue of making our testing framework more complicated,
+but committing the files leads to a potential situation where we need to regenerate the
+files ad hoc. For the initial implementation of our testing framework we are choosing to
+create the files and commit them to the repository. The below python code was used
+to generate the first versions of these datasets.
```

Comment:
why don't you add this as a regular script here and refer to it? If I want to run it myself then I have to copy that out now :(